CONTRIBUTION.md

```diff
@@ -0,0 +1,161 @@
+
+### AsciiDoc Formatting
+Eclipse Che documentation use AsciiDoc for formatting. See [AsciiDoc Writer's Guide](https://asciidoctor.org/docs/asciidoc-writers-guide/) for syntax and general help with AsciiDoc.
+
+#### Using `test-adoc` for AsciiDoc syntax verification
+To verify that the syntax of your document is correct, use [`test-adoc`](https://github.com/jhradilek/check-links) script. 
+`test-adoc.sh` is integrated into the `run.sh` script in the root directory of the project.
+ 
+To run `test-adoc`, execute from the root directory of the project: 
+```
+$ bash run.sh -test-adoc <PATH_TO_THE_FILE>
+```
+
+### Documentation Structure
+Eclipse Che documentation follow [Modular Documentation initiative](https://redhat-documentation.github.io/modular-docs/). Use templates provided in the [Modular Documentation Reference](https://redhat-documentation.github.io/modular-docs/#creating-modules) to create new user stories.
+
+To add a new page, create an `.adoc` file in `src/main/pages/che-<version>/<directory_name>` 
+
+<b id="f2">Versions: </b>
+* `che-6` - Eclipse Che 6 content. Note that Che 6 is deprecated. 
+* `che-7` - Eclipse Che 7 content, including y-streams and z-streams.   
+
+<b id="f1">Available directories: </b> 
+* `overview` - introductory section
+* `end-user-guide` - documentation for developers: navigating dashboard, working in Che-Theia, and so on 
+* `installation-guide` - installation guides
+* `administration-guide` - documentation for administrators of the clusters: configuring Che on a cluster, managing users, monitoring resources, security and data recovery 
+* `contributor-guide` - how to develop plug-ins for Che, add new debuggers, languages, and so on
+* `extensions` - documentation about extensions for Che, such as Eclipse Che4z, OpenShift Connector 
+
+#### Using `newdoc` to create templates for new documents
+
+With a [`newdoc`](https://github.com/redhat-documentation/tools/tree/master/newdoc) script, you can generate empty modular templates for your new user story.  
+`newdoc` is integrated into a `run.sh` script in the root directory of the project.
+
+To run `newdoc`, execute from the root directory of the project:  
+```
+$ bash run.sh -newdoc --procedure "Title of your new procedure file"
+```
+
+### Writing documentation
+#### Style
+Eclipse Che documentation follows the The IBM Style Guide. If you do not have a paper copy of the styleguide, you can refer to  
+[developerWorks editorial style guide](https://www.ibm.com/developerworks/library/styleguidelines/index.html) on the IBM website. While `developerWorks` is not longer supported, it still provides useful reference information. 
+
+#### Using `vale` to check the style
+[`vale`](https://errata-ai.gitbook.io/vale/) is a command-line linter. With `vale`, you can check if the style of your documentation follows [the language rules and recommendations defined in Eclipse Che project](https://github.com/eclipse/che-docs/tree/master/.ci/vale/styles).
```

Comment:
"defined in the Eclipse Che project"

Also, should this be "the Eclipse Che documentation project"?